src/kinetics/ImplicitSurfChem.cpp

```diff
@@ -107,6 +112,18 @@ void ImplicitSurfChem::getState(doublereal* c)
 void ImplicitSurfChem::initialize(doublereal t0)
 {
     m_integ->setTolerances(m_rtol, m_atol);
+    if (m_maxstep > 0)
+    {
+        m_integ->setMaxStepSize(m_maxstep);
+    }
```

Comment:
Given the call to `setMaxStepSize` in `ImplicitSurfChem::integrate`, this has no effect, right?